pkg/k8sops/k8sops.go

```diff
@@ -67,16 +70,15 @@ func New(kubeCfgFile, masterHost string) (*K8sops, error) {
 
 func buildConfig(kubeCfgFile string) (*rest.Config, error) {
 	if kubeCfgFile != "" {
-		logrus.Infof("using OutOfCluster k8s config with kubeConfigFile: %s", kubeCfgFile)
+		fmt.Println(fmt.Sprintf("using OutOfCluster k8s config with kubeConfigFile: %s", kubeCfgFile))
 		config, err := clientcmd.BuildConfigFromFlags("", kubeCfgFile)
 		if err != nil {
 			panic(err.Error())
```

Comment:
similar to fatalf, don't panic like this either - bubble up the error to the top level

Reply:
Done